OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -63,6 +63,8 @@ public class ArmamentInfo : ConditionalTraitInfo, Requires<AttackBaseInfo>
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		public readonly bool ShootAtClosestTargetablePosition = false;
```

Comment:
Iirc cell based behavour has been a wanted feature for quite some time. Infiltration behaviour included. This is a gameplay feature decition I like to seperate a bit for balancing but IMO at worst it's one step back traded for two steps forward. :+1: